src/Http/Http.Abstractions/src/Microsoft.AspNetCore.Http.Abstractions.csproj

```diff
@@ -19,6 +19,7 @@ Microsoft.AspNetCore.Http.HttpResponse</Description>
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Http.Features" />
     <Reference Include="Microsoft.Net.Http.Headers" />
+    <Reference Include="Microsoft.Extensions.Logging.Abstractions" PrivateAssets="All" />
```

Comment:
?

Reply:
I had to add a reference for the source generator. Our build complains if you add new references and I didn't want to deal with it.